src/App.jsx

```diff
@@ -1,10 +1,53 @@
-import { useState } from 'react'
+import { useState, useEffect } from 'react';
 import './App.css'
+import MovieList from './MovieList'
+import Header from './Header'
+import Footer from './Footer'
 
 const App = () => {
-  <div className="App">
-    
+  const [loadedMovies, setLoadedMovies] = useState([]);
+  const [page, setPage] = useState(1); //Page number of next page to load
+  const baseURL = "https://image.tmdb.org/t/p/w500";
```

Comment:
1. Static constants like this you want to define at the top of the file. That way it doesn't need to redefined in every render
2. Seems like this constant is actually unused so would want to clean it up